### PR TITLE
Allow storing of other tokens and cross chain token transfers

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: runtime/neumann/Cargo.toml [hunk 1]
```diff
@@ -91,6 +91,14 @@ xcm-executor = { git = "https://github.com/paritytech/polkadot", default-feature
 # Moonbeam Dependencies
 parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "oak-polkadot-v0.9.19" }
 
+# ORML Dependencies
```
Comment (irsal) [inline runtime/neumann/Cargo.toml:94]: another version dependency to keep track of 🙃

**Comment by rhuttman** [comment]
Tested locally:
- Transferring Kusd, KAR, LKSM, and TUR between chains

File: runtime/neumann/src/xcm_config.rs [hunk 2]
```diff
@@ -166,19 +173,70 @@ pub type Barrier = DenyThenTry<
...
+        if let MultiAsset {
+            id: AssetId::Concrete(id),
+            fun: Fungibility::Fungible(amount),
+        } = revenue
+        {
+            if let Some(currency_id) = CurrencyIdConvert::convert(id) {
+                // Ensure TreasuryAccount have ed requirement for native asset, but don't need
+				// ed requirement for cross-chain asset because it's one of whitelist accounts.
+				// Ignore the result.
+				let _ = Currencies::deposit(currency_id, &TreasuryAccount::get(), amount);
```
Comment (nvengal) [inline runtime/neumann/src/xcm_config.rs:219]: nit: formatting

File: runtime/turing/src/xcm_config.rs [hunk 3]
```diff
@@ -165,19 +172,106 @@ pub type Barrier = DenyThenTry<
...
+        if let MultiAsset {
+            id: AssetId::Concrete(id),
+            fun: Fungibility::Fungible(amount),
+        } = revenue
+        {
+            if let Some(currency_id) = CurrencyIdConvert::convert(id) {
+                // Ensure TreasuryAccount have ed requirement for native asset, but don't need
+				// ed requirement for cross-chain asset because it's one of whitelist accounts.
+				// Ignore the result.
+				let _ = Currencies::deposit(currency_id, &TreasuryAccount::get(), amount);
```
Comment (nvengal) [inline runtime/turing/src/xcm_config.rs:247]: nit: formatting.

Is it worth logging or something if this fails here? This seems like the sort of thing that we wouldn't just want to fail silently.

**Review by nvengal — APPROVED** [review]

File: runtime/neumann/src/xcm_config.rs (hunk 2 above)
Comment (rhuttman) [reply runtime/neumann/src/xcm_config.rs:219]: believe it or not the format looks correct in my IDE, so not sure what is going on here

File: runtime/turing/src/xcm_config.rs (hunk 3 above)
Comment (rhuttman) [reply runtime/turing/src/xcm_config.rs:247]: I need to think about how to handle these sorts of things. Will add later this week

**Pull request merged**